src/main/java/com/xebialabs/overcast/OvercastProperties.java

```diff
@@ -89,6 +89,10 @@ public static String getOvercastProperty(String key) {
         return getOvercastProperty(key, null);
     }
 
+    public static List<String> getOvercastProperties(String key) {
+        return getOvercastProperties(key, (String)null);
```

Comment:
Hmm do I see correctly that if the key is not there a non empty list with a null value is returned? Would it make more sense to return an empty list or a null (ugly but more consistent with how the other ones work)?